metadata.yaml

```diff
@@ -1,16 +1,23 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 name: mysql-router-k8s
-display-name: MySQL Router
-maintainers:
-  - Carl Csaposs <carl.csaposs@canonical.com>
-  - Paulo Machado <paulo.machado@canonical.com>
-  - Shayan Patel <shayan.patel@canonical.com>
+display-name: MySQL Router K8s
```

Comment:
non blocker: remind me, should we use "Charmed <thing>" on every name?

Reply:
I do not have an answer. "Charmed" got a marketing meaning... on product level.
"Charmed MySQL" is a product AND charm name simultanously.

Product "Charmed MySQL" contains charm "Charmed MySQL" + charm "MySQL Router" + ...
This is why I didn't use Charmed for MySQL Router, as it is not a product, it is a peace...

List of products https://ubuntu.com/data/
P.S. I can be wrong :-D